pandas/tests/indexing/test_timedelta.py

```diff
@@ -47,3 +48,14 @@ def test_string_indexing(self):
         expected = df.iloc[0]
         sliced = df.loc['0 days']
         tm.assert_series_equal(sliced, expected)
+
+    @pytest.mark.parametrize(
+        "value, expected",
+        [(None, [pd.NaT, 1, 2]),
```

Comment:
you don't need to pass expected here as they are the same, just define in the body